scripts/copy_nwb_strip_processing.py

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env python
+import argparse, glob, os
+from pynwb import NWBHDF5IO
+
+
+parser = argparse.ArgumentParser(description='Copies nwb and strips away processing module')
+parser.add_argument('src_file', type=str, help='Source nwb file')
+parser.add_argument('--dst_file', type=str, default=None,
```

Comment:
I'm a bit wary of having the default option be to overwrite nwb files. IMO, users should have to specify that they want to overwrite things.